HTML/QuickForm/group.php

```diff
@@ -97,13 +97,15 @@ class HTML_QuickForm_group extends HTML_QuickForm_element
      * @param     bool      $appendName     (optional)whether to change elements' names to
      *                                      the form $groupName[$elementName] or leave
      *                                      them as is.
+     * @param     mixed     $attributes     (optional)Either a typical HTML attribute string
+     *                                      or an associative array. Date format is passed along the attributes.
      * @since     1.0
      * @access    public
      * @return    void
      */
-    function __construct($elementName=null, $elementLabel=null, $elements=null, $separator=null, $appendName = true)
+    function __construct($elementName=null, $elementLabel=null, $elements=null, $separator=null, $appendName = true, $attributes = null)

```

Comment:
Spot-checked: I grepped for examples of `group`. Picked one from "Administer => CiviContribute => Premiums => ... => Edit => Premium Image" (a group of radio buttons). It still works in PHP 5.3.